### PR TITLE
Add Balaibalan, Láadan, Lingua Franca Nova, Lojban, Toki Pona to language list

### LINKED ISSUE #8995: Add support for more constructed languages in "Posting language" dropdown menu
Hello all,

Only Esperanto is available, but I would like Mastodon to add support for more constructed languages as: Bâleybelen, Ido, Interlingua, Láadan, Lingua Ignota, Lingwa de Planeta, Mänti, Na'vi, Toki Pona, Volapük, etc.

I speak one of these constructed languages and currently it is hard to find another people at Mastodon tooting in the same language, because they're not available at Mastodon for the time being.

Thank you so much for reading!

### PR DESCRIPTION
**Pull request by VyrCossont**
Fixes #8995.

### CODE REVIEW

**Comment by VyrCossont** [comment]
These linter settings seem a bit opinionated. I'm following existing style in that file exactly.

File: app/helpers/languages_helper.rb [hunk 1]
```diff
@@ -189,8 +189,12 @@ module LanguagesHelper
   ISO_639_3 = {
     ast: ['Asturian', 'Asturianu'].freeze,
     ckb: ['Sorani (Kurdish)', 'سۆرانی'].freeze,
+    jbo: ['Lojban', 'Lojban'].freeze,
```
Comment (tbodt) [inline app/helpers/languages_helper.rb:192]: The name for Lojban in Lojban is la .lojban.

File: app/helpers/languages_helper.rb [hunk 2]
```diff
@@ -189,8 +189,12 @@ module LanguagesHelper
...
     kab: ['Kabyle', 'Taqbaylit'].freeze,
     kmr: ['Kurmanji (Kurdish)', 'Kurmancî'].freeze,
+    ldn: ['Láadan', 'Láadan'].freeze,
+    tok: ['Toki Pona', 'Toki Pona'].freeze,
```
Comment (tbodt) [inline app/helpers/languages_helper.rb:196]: Should be lowercase (`toki pona`)

File: app/helpers/languages_helper.rb (hunk 1 above)
Comment (VyrCossont) [reply app/helpers/languages_helper.rb:192]: Thank you, I'll fix that.

File: app/helpers/languages_helper.rb (hunk 2 above)
Comment (VyrCossont) [reply app/helpers/languages_helper.rb]: Already fixed.

**Comment by VyrCossont** [comment]
And it looks like I accidentally truncated the translated name for Balaibalan, which has now been fixed.

**Review by tbodt — APPROVED** [review]

**Comment by ykzts** [comment]
There must be sufficient translations available after enabling the translation in [Crowdin](https://crowdin.com/project/mastodon) first.

**Comment by VyrCossont** [comment]
@ykzts These are not intended for UI translations, but for post language tagging and language filtering, so translation availability is irrelevant. Conlangs already in Mastodon's language map and _not_ accompanied by translations in Crowdin include both Interlingua and Interlingue, and there are numerous _natural_ languages in that list that have no translations in Crowdin, from Old Church Slavonic to Xhosa.

**Comment by ykzts** [comment]
Ah, yes, you are correct as the `available_locales` has not been updated.

**Review by ykzts — APPROVED** [review]

**Comment by VyrCossont** [comment]
I'm going to suggest that we disable those two specific Rubocop checks: `Metrics/ModuleLength` because this is basically tabular data and the module length is not an indicator of code complexity, and `Style/WordArray` because it doesn't make sense to mix `%w`/`%W` arrays of bare words with normal arrays of `["quoted phrases"]` (which would significantly change the existing formatting of the file).

**Comment by VyrCossont** [comment]
No other linter issues. Should be ready to merge. Thanks for the reviews, @ykzts and @tbodt.

**Pull request merged**